module2/homework/addEven/addEven.hpp

```diff
@@ -1,8 +1,14 @@
 #pragma once
 #include <vector>
 
-int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+int addEven(const std::vector<int>& numbers) 
+{
+    int result = 0;
+
+    for (auto number : numbers)
```

Comment:
```suggestion
    for (const auto & number : numbers)
```
You do not need to modify the ```number``` so you can make it ```const &```

Reply:
Changed. Thx!